### PR TITLE
[ZEPPELIN-2672] Fix interpreter repos url tooltip

### PR DESCRIPTION
**Pull request by necosta**
### What is this PR for?
Fixing regression on showing interpreter repos tooltip

### What type of PR is it?
Bug Fix

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2672

### How should this be tested?
Zeppelin > Interpreter > Repository > (Mouse over) central/local/other

### Screenshots 
After this fix:
image

### Questions:
* Does the licenses files need update? N
* Is there breaking changes for older versions? N
* Does this needs documentation? N

### CODE REVIEW

File: zeppelin-web/src/app/interpreter/interpreter.html [hunk 1]
```diff
@@ -62,15 +62,7 @@ <h3 class="new_h3">
       <p>Available repository lists. These repositories are used to resolve external dependencies of interpreter.</p>
       <ul class="noDot">
         <li class="liVertical" ng-repeat="repo in repositories">
-          <a tabindex="0" class="btn btn-info" role="button"
-             popover-trigger="focus"
```
Comment (1ambda) [inline zeppelin-web/src/app/interpreter/interpreter.html]: Can we replace all popover angualr directives?

Comment (necosta) [inline zeppelin-web/src/app/interpreter/interpreter.html]: Yes. There was just one more. Fixed in latest commit.

Comment (1ambda) [inline zeppelin-web/src/app/interpreter/interpreter.html]: Then I think we can remove the popover directive related files as well.

Comment (necosta) [inline zeppelin-web/src/app/interpreter/interpreter.html]: Done

Comment (1ambda) [inline zeppelin-web/src/app/interpreter/interpreter.html]: Checked tooltip now works well. +1
But the tooltip for the first repo button is trimmed. Can we set tooltip position for repo buttons?

image

File: zeppelin-web/src/app/tabledata/columnselector_settings.html [hunk 2]
```diff
@@ -47,11 +47,10 @@
               style="font-weight: 300; font-size: 13px; margin-left: 1px;"
               ng-style="prop.name === 'xAxis' ? { 'background-color': '#906ebd' } : prop.name === 'yAxis' ? { 'background-color': '#cd5c5c' } : prop.name === 'group' ? { 'background-color': '#5782bd' } : ''">
         {{prop.name}}
-        <a tabindex="0" class="fa fa-info-circle" role="button" popover-placement="top"
+        <a tabindex="0" class="fa fa-info-circle" role="button"
            ng-if="prop.tooltip"
            style="color: white;"
-           popover-trigger="focus"
-           popover-html-unsafe="{{prop.tooltip}}"></a>
+           uib-tooltip="{{prop.tooltip}}"></a>
```
Comment (1ambda) [inline zeppelin-web/src/app/tabledata/columnselector_settings.html]: I think we need to use `uib-tooltip-html` 

- https://angular-ui.github.io/bootstrap/#!#tooltip

image

**Comment by 1ambda** [comment]
Checked that works well. LGTM except for 2 comments i left.

**Comment by 1ambda** [comment]
Hi, @necosta

**Comment by necosta** [comment]
Yes, @1ambda , expect updates later today. Have a few challenges on the column selector script...

**Comment by necosta** [comment]
@1ambda , ready for review. Thanks!

File: zeppelin-web/src/app/visualization/builtins/visualization-scatterchart.js [hunk 3]
```diff
@@ -34,12 +34,12 @@ export default class ScatterchartVisualization extends Nvd3ChartVisualization {
       },
       {
         name: 'size',
-        tooltip: `<li>Size option is valid only when you drop numeric field here.</li>
-                  <li>When data in each axis are discrete,
-                  'number of values in corresponding coordinate' will be used as size.</li>
-                  <li>Zeppelin consider values as discrete when the values contain string value
-                  or the number of distinct values are bigger than 5% of total number of values.</li>
-                  <li>Size field button turns to grey when the option you chose is not valid.</li>`
+        tooltip: `This option is only valid for numeric fields.
```
Comment (necosta) [inline zeppelin-web/src/app/visualization/builtins/visualization-scatterchart.js]: To me, it looked this message could be improved. Any suggestions, please let me know. I'm not sure my new sentence is clear as well :)

Comment (1ambda) [inline zeppelin-web/src/app/visualization/builtins/visualization-scatterchart.js]: Looks good to me. It's more concise.

File: zeppelin-web/src/app/visualization/builtins/visualization-scatterchart.js [hunk 4]
```diff
@@ -34,12 +34,12 @@ export default class ScatterchartVisualization extends Nvd3ChartVisualization {
...
+          When data in each axis is discrete,
```
Comment (1ambda) [inline zeppelin-web/src/app/visualization/builtins/visualization-scatterchart.js]: This too.

**Pull request closed without merging**